src/data/password-manager.json

```diff
@@ -17,15 +17,28 @@
         "nextSteps": [
             {
                 "prompt": "Oh no! Ok, I'll use a password manager.",
-                "step": "install"
+                "step": "where"
             },
             {
                 "prompt": "Still seems to hard.",
                 "step": "aww"
             }
         ]
     },
-    "install": {
+    "where": {
+        "description": "Do you wanna install the password manager in your browser and have it sync with your phone for easy access, or only be able to access it locally your computer?",
```

Comment:
Could you change the wording there? I like the style be more like "Here are the two options, which do you like?" 

Maybe something like:
> There are a few options for password managers! You could install one in your browser and have it sync with your phone for easy access, or keep your passwords extra safe by storing them locally your computer.